### ISSUE TITLE
Some viewpoints in the path are not reachable?

### ISSUE THREAD

**Issue by cswhjiang** [peteanderson80/Matterport3DSimulator#25]
It seems that for the scan=`8WUmhLawc2A`, instr_id=`6825_2`, the robot from viewpoint `550d66ef28114bef8525d3a2d6db9cd2` can not reach the next viewpoint `01b439d39a8f412fa1837be7afb45254` by adjusting heading and elevation. The viewpoint `01b439d39a8f412fa1837be7afb45254` might be in the lists of navigableLocations of 36 combinations of heading and elevation, but the indexes are never 1. Hence, it can not be reached by adjusting the angles of heading and elevation and moving forward.

**Comment by peteanderson80** [comment]
Thanks @cswhjiang, due to the irregular nature of the navigation graphs, we suspected that this might happen in a few examples when using the simplified instruction set implemented in our baselines. 
However, we do not intend that all models should have to use discretized heading and elevation, or a single monolithic 'forward' instruction. In other words, the agent is allowed to select any navigableLocation, we just took a very simple approach in our baselines.
